deisctl/cmd/cmd.go

```diff
@@ -184,13 +184,19 @@ func startDefaultServices(b backend.Backend, wg *sync.WaitGroup, outchan chan st
 	wg.Wait()
 
 	// optimization: start all remaining services in the background
+	// cache is a special case because it must start before registry
+	b.Start([]string{"cache"}, &_wg, _outchan, _errchan)
```

Comment:
I was also confused why this whole block even exists. Rather than remove/refactor it out now, I just modified it to do what we want with cache. I confirmed that cache is started, we wait for it to come up, then we resume. I was going to ask @gabrtv why the block exists - he implemented it, according to git blame.